src/components/side-nav/side-nav.js

```diff
@@ -14,23 +14,26 @@ import {
 import { NavLink } from "react-router-dom"
 
 const SideNav = () => {
+    const handleActiveNav = (navigationData) => {
+        return navigationData.isActive ? styles.active : ""
+    }
     return (
         <aside className={styles.container}>
             <List style="none">
                 <ListItem className={styles.nav_link}>
-                    <NavLink className={(navigationData) => navigationData.isActive ? styles.active : null} to="/explore"><IconButton icon={<MdOutlineExplore />} className={styles.icon} /></NavLink>
```

Comment:
missing prettier / code formatter, line length crossing 80

Reply:
yeah not sure why it does that